dspace-uclouvain/src/main/java/org/dspace/uclouvain/packager/DSpaceUCLouvainMETSIngester.java

```diff
@@ -2,23 +2,68 @@
 
 import org.apache.logging.log4j.Logger;
 import org.dspace.authorize.AuthorizeException;
+import org.dspace.content.Bitstream;
 import org.dspace.content.DSpaceObject;
 import org.dspace.content.Item;
+import org.dspace.content.MetadataField;
+import org.dspace.content.authority.Choices;
 import org.dspace.content.crosswalk.CrosswalkException;
 import org.dspace.content.crosswalk.MetadataValidationException;
-import org.dspace.content.packager.*;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.packager.AbstractMETSIngester;
+import org.dspace.content.packager.DSpaceMETSIngester;
+import org.dspace.content.packager.METSManifest;
+import org.dspace.content.packager.PackageParameters;
+import org.dspace.content.packager.PackageValidationException;
+import org.dspace.content.service.BitstreamService;
+import org.dspace.content.service.MetadataFieldService;
 import org.dspace.core.Constants;
 import org.dspace.core.Context;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
 import org.dspace.workflow.WorkflowException;
+import org.jdom2.Content;
+import org.jdom2.Element;
+import org.jdom2.transform.JDOMResult;
+import org.jdom2.transform.JDOMSource;
 
+import javax.xml.transform.*;
+import javax.xml.transform.stream.StreamSource;
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.sql.SQLException;
+import java.util.*;
```

Comment:
Same here, use literal import for `java.util.List` && `java.util.ArrayList` instead of all `java.util.*`.

Reply:
Same correction than above, this does not really matter.